rts/lra/lra-test/tck/pom.xml

```diff
@@ -168,6 +167,8 @@
                                         <argument>-Djava.net.preferIPv4Stack=true</argument>
                                         <argument>${lra.coordinator.debug.params}</argument>
                                         <argument>${lra.coordinator.trace.params}</argument>
+                                        <argument>-DRecoveryEnvironmentBean.periodicRecoveryPeriod=10</argument>
```

Comment:
this might be too short in CI, I guess we will find out

Reply:
It seems to work fine (on CI and the laptop) - the goal is to minimise the time on the TCK tests (since we no longer have a "trigger recovery mechanism")

Reply:
Also it is only for testing against the LRA axis where we use arquillian to launch a thorntail MP container so will not affect any other jobs.